api/src/hyp3_api/openapi-spec.yml

```diff
@@ -207,6 +207,10 @@ components:
         - RTC_GAMMA
 
     job_parameters:
+      oneOf:
+        - $ref: "#/components/schemas/rtc_gamma_job_parameters"
```

Comment:
ideally we'd enforce that "`job_type": "RTC_GAMMA"` goes with `rtc_gamma_job_parameters and `"job_type": "INSAR_GAMMA" goes with `insar_gamma_job_parameters`, but I'm comfortable punting that until later